_specs/ecip-proto-treasury-system.md

```diff
@@ -0,0 +1,450 @@
+---
+lang: en
+ECIP: ECIP-XXX
```

Comment:
Also, can you please format the header as it is formatted in the template file? [ecip-X.md](https://github.com/ethereumclassic/ECIPs/blob/master/ecip-X.md)

Otherwise, it will not render on the website.

Reply:
It should match now. :)